Jenkinsfile-bundle

```diff
@@ -0,0 +1,64 @@
+library("tdr-jenkinslib")
+
+def versionTag = "v${env.BUILD_NUMBER}"
+
+pipeline {
+  agent {
+    label "master"
+  }
+  parameters {
+    text(name: "BUILD_IMAGE_TAG", defaultValue: "latest", description: "The image tag for the base yara images")
+  }
+  stages {
+    stage("Build zip file") {
+      steps {
+        script {
+          sh "aws ecr get-login --region eu-west-2 --no-include-email | bash"
+          sh "docker stop dependencies | true"
+          sh "docker stop rules | true"
```

Comment:
This might be a silly question, but why do we need to run `docker stop` during a build? Why would there be any Docker containers running at this point?

Reply:
It does need to be run because this is running on the master Jenkins container, the container persists between builds so we do need to run `docker stop` to delete them. It doesn't really belong here though so I've moved it to a post block so it's cleaned up after each run. 
I've also put the code to remove the lambda folder in there as this can persist if the build fails.

Reply:
Cheers, I think that's easier to understand. 👍